sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class Delay
+    {
+        protected Delay(System.TimeSpan? maxDelay = default(System.TimeSpan?), double jitterFactor = 0.2) { }
+        public static Azure.Core.Delay CreateExponentialDelay(System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.Delay CreateFixedDelay(System.TimeSpan? delay = default(System.TimeSpan?)) { throw null; }
+        public System.TimeSpan GetNextDelay(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        public System.Threading.Tasks.ValueTask<System.TimeSpan> GetNextDelayAsync(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        protected abstract System.TimeSpan GetNextDelayCore(Azure.Response? response, int retryNumber, System.Collections.Generic.IDictionary<string, object?> context);
+        protected abstract System.Threading.Tasks.ValueTask<System.TimeSpan> GetNextDelayCoreAsync(Azure.Response? response, int retryNumber, System.Collections.Generic.IDictionary<string, object?> context);
```

Comment:
It might be overkill to offer an async version of calculating how long to wait next.

Reply:
Yes, that is fair, but we offer the async version in the RetryPolicy since it derives from HttpPipelinePolicy. If we left the async off for now, we would not be able to be add it as abstract later if we need to, so I would err towards leaving it.